plugin/src/main/scala/higherkindness/mu/rpc/srcgen/SrcGenPlugin.scala

```diff
@@ -31,7 +31,7 @@ import scala.concurrent.ExecutionContext.global
 
 object SrcGenPlugin extends AutoPlugin {
 
-  override def trigger: PluginTrigger = allRequirements
+  override def trigger: PluginTrigger = noTrigger
```

Comment:
open question for @juanpedromoreno or @cb372; I'd love some clarification about the release process here.

Reply:
The release process is initiated by using Git tags/releases. The release notes are automatically populated based on the [labels](https://github.com/higherkindness/sbt-mu-srcgen/blob/master/.github/labels.yml) (that's why I added the `breaking change` to this PR).

Example: https://github.com/higherkindness/mu-scala/releases/tag/v0.22.0

Reply:
In this particular case, I wouldn't release anything yet, since we might need to add some more breaking changes in the mu-scala side so that we could make a joint release, including all the breaking changes at once.

Reply:
oh okay cool; thanks for the insights about the releases!  I love how automated it all is :)  And yes, today I'm making the changes to `mu-scala` to address the changed behavior of `mu-sbt-srcgen` and I think a joint release makes the most sense to me.

Reply:
@juanpedromoreno is there a wiki on which labels we use for different release types?

Reply:
Some PRs, like the scala-steward or @47erbot ones, are automatically labeled. For this kind of PRs, features, or documentation ones, they need to be manual, .. and unfortunately, we don't have any wiki for that yet.

Reply:
got it, thanks for clarifying.  This plus our offline convo has me up to speed on everything.